Automattic/wp-calypso
# Update Help Center Success Icon

Enhancement to update the design of the Help Center submission success icon.

## Proposed Changes

* Updates the SVG to our current primary color and replaces a shadow circle with a full background color

## Testing Instructions



* Open Help Center
* Click Still Need Help
* Continue to contact us and choose the email option
* Submit the form and see the updated design

Old:

![Screenshot 2024-02-29 at 7 59 04 AM](https://github.com/Automattic/wp-calypso/assets/1900444/db196340-10a9-4c4a-b259-be07de2d0ad1)

New:

![Screenshot 2024-02-29 at 12 37 28 PM](https://github.com/Automattic/wp-calypso/assets/1900444/b2aae0d1-6c83-4919-af94-c7c708c25715)

## Pre-merge Checklist
